.github/workflows/continous-integration.yml

```diff
@@ -1095,8 +1105,6 @@ jobs:
           IS_NEWEST_VERSION=${{ needs.build_docker_base_images_and_set_env.outputs.is_newest_version }}
 
           docker buildx bake --set *.platform=linux/amd64,linux/arm64 -f docker/docker-bake.hcl ${{ matrix.image }} --push
-          docker buildx bake --set *.platform=linux/amd64 -f docker/docker-bake.hcl ${{ matrix.image }} --load
```

Comment:
`--load` was added to overcome this https://rasa-hq.slack.com/archives/C01H2UCNP2T/p1688465112595979?thread_ts=1688458013.078029&cid=C01H2UCNP2T issue. Hopefully this wont recur.

Reply:
The original issue mentioned in the Slack thread was because we were using buildx to build the image and that image was stored in a Docker registry inside a docker container which runs buildkit.
But when docker tag is invoked, it is working with a different docker registry, one running as part of the Docker Engine on the worker on GitHub.

You can move image from one Docker registry to another with `--load` flag for buildx.
Now there seems to be the bug when we want to move images for amd64 and arm64 from buildx's docker registry to docker registry for GitHub's docker engine.

To fix that we are building and tagging multi-platform image inside the buildx's docker registry and pushing it from that registry to our official docker registry.